src/solver/hydro/management/management.cpp

```diff
@@ -168,6 +168,16 @@ void HydroManagement::prepareNetDemand(uint numSpace)
                 - ((ModeT != Data::stdmAdequacy) ? scratchpad.mustrunSum[hour]
                                                  : scratchpad.originalMustrunSum[hour]);
 
+            // RES
```

Comment:
In the comment, RES is too generic, I would mention clusters

Reply:
Agreed